app/src/components/Button.css

```diff
@@ -0,0 +1,28 @@
+button {
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 25px;
+  letter-spacing: 0.2px;
+  display: inline-flex;
+  height: 50px;
+  width: 280px;
```

Comment:
lets not fix `width` for the button

cause depending on the text provided for the button in can be longer or lesser than `280px`, right?

just delete it and add className for button. in the `Card.jsx` component you can apply `width: 100%` so it is aligned in the container for full width. 

all the same as comment above for `Avatar.jsx` algorithm of actions